envs/cytominer_env.yaml

```
@@ -0,0 +1,10 @@
+name: cytominer_env
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python>=3.7.0
+  - pip
+  - pip:
+    - "pycytominer"
```

```suggestion
  - conda-forge::pycytominer=0.1

```

Wooo! pycytominer is on conda 💪 https://anaconda.org/conda-forge/pycytominer